src/include/network/widgets.rb

```diff
@@ -152,6 +152,7 @@ def ValidateIP(key, _event)
     end
 
     def handleStartmode(_key, _event)
+      # FIXME: not working with object CWM
```

Comment:
or better, it can work, but it need to store ifplugd_priority widget and operate on them and not direcly on UI ChangeWidget level.